crawler/priority_samples_process.py

```diff
@@ -102,68 +124,65 @@ def query_any_unprocessed_samples(db: Database) -> List[SamplePriorityDoc]:
-    """Update sample records in the MLWH database from unprocessed priority samples,
-    including the corresponding mongodb _id, must_seqequence, preferentially_sequence
+    logger.info("Updating Mongodb priority samples to processed")
+    # use stored identifiers to update priority_samples table to processed true
+    sample_ids = list(map(extract_sample_id, samples))
 
-    Arguments:
-        samples {List[Any]} -- List of unprocessed priority samples
+    priority_samples_collection = get_mongo_collection(db, COLLECTION_PRIORITY_SAMPLES)
+    for sample_id in sample_ids:
+        priority_samples_collection.update({FIELD_SAMPLE_ID: sample_id}, {"$set": {FIELD_PROCESSED: True}})
 
-    Returns:
-        {bool} -- True if the insert was successful; otherwise False
-    """
-    values = list(map(map_mongo_sample_to_mysql, samples))
+    logger.info("Mongo update of processed for priority samples successful")
 
-    mysql_conn = create_mysql_connection(config, False)
 
-    if mysql_conn is not None and mysql_conn.is_connected():
-        try:
-            run_mysql_executemany_query(mysql_conn, SQL_MLWH_MULTIPLE_INSERT, values)
+def logging_message_object() -> Dict:
+    return {
+        "success": {
+            "msg": "MLWH database inserts completed successfully for priority samples",
+        },
+        "insert_failure": {
+            "error_type": "TYPE 28",
+            "msg": "MLWH database inserts failed for priority samples",
+            "critical_msg": "Critical error while processing priority samples'",
+        },
+        "connection_failure": {
+            "error_type": "TYPE 29",
+            "msg": "MLWH database inserts failed for priority samples, could not connect",
+            "critical_msg": "Error writing to MLWH for priority samples, could not create Database connection",
+        },
+    }
 
-            logger.debug("MLWH database inserts completed successfully for priority samples")
-            return True
-        except Exception as e:
-            logging_collection.add_error(
-                "TYPE 28",
-                "MLWH database inserts failed for priority samples",
-            )
-            logger.critical(f"Critical error while processing priority samples': {e}")
-            logger.exception(e)
-    else:
-        logging_collection.add_error(
-            "TYPE 29",
-            "MLWH database inserts failed for priority samples, could not connect",
-        )
-        logger.critical("Error writing to MLWH for priority samples, could not create Database connection")
 
-    return False
+def update_priority_samples_into_mlwh(samples: List[Any], config: Config) -> bool:
+    mlwh_success = insert_or_update_samples_in_mlwh(samples, config, True, logging_collection, logging_message_object())
+    if mlwh_success:
+        logger.info("MLWH insert successful")
+    return mlwh_success
 
 
-# TODO: refactor duplicated function insert_plates_and_wells_from_docs_into_dart in file_processing.py
```

Comment:
This comment is still helpful if it's something worth doing in the future? This does look similar to the methoed we already use for DART.